File: src/main/java/killa/modelo/DetallePedido.java
package killa.modelo;

public class DetallePedido {
    private int id_detallePedido;
    private int cantidad;
    private double precioAplicado;

    // Navegabilidad
    private Pedido pedido;
    private Producto producto;

    public DetallePedido() {}
}

File: FrontEnd/BlazorAppKillaBeauty/BlazorAppKillaBeauty/ClienteREST/Models/DetallePedido.cs
using System.Text.Json.Serialization;

namespace BlazorAppKillaBeauty.ClienteREST.Models
{
    public class DetallePedido
    {
        [JsonPropertyName("cantidad")]
        public int Cantidad { get; set; }

        [JsonPropertyName("precioAplicado")]
        public double PrecioAplicado { get; set; }

        [JsonPropertyName("producto")]
        public ProductoRef? Producto { get; set; }
    }
}
